### ISSUE TITLE
Support `Deal.find_by_contact(contact)`

### ISSUE THREAD

**Issue by tommotaylor** [adimichele/hubspot-ruby#127]
I proposal to add support for finding a deal by its contact. Eg:

```
 def find_by_contact(contact)
  path = ASSOCIATED_DEAL_PATH
  params = { objectType: :contact, objectId: contact.vid }
  response = Hubspot::Connection.get_json(path, params)
  response["results"].map { |deal_id| find(deal_id) }
end
```

Its almost a copy and paste of https://github.com/adimichele/hubspot-ruby/blob/master/lib/hubspot/deal.rb#L70

**Comment by cbisnett** [comment]
If you want to turn this into a PR I'll merge it. Otherwise I can add it since it's quite simple.

**Comment by cbisnett** [comment]
This was implemented and merged in #82.
